cmd/chihaya/config.go

```diff
@@ -63,6 +72,16 @@ func ParseConfigFile(path string) (*ConfigFile, error) {
 	return &cfgFile, nil
 }
 
+// NewPeerStore allocates a storage.PeerStore for a given ConfigFile.
+func (cfg ConfigFile) NewPeerStore() (storage.PeerStore, error) {
+	storecfg := cfg.MainConfigBlock.Storage
+	ps, exists, err := storage.NewPeerStore(storecfg.Name, storecfg.Config)
```

Comment:
Huh, I would've expected something like an `ErrUnknownDriver`, but this works too. Try something new from time to time :)